networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/libp2p/LibP2PPeer.java

```diff
@@ -89,11 +89,13 @@ public void checkPeerIdentity() {
     getAgentVersionFromIdentity()
         .thenAccept(
             maybeAgent -> {
-              LOG.debug("Connected peer has agent string: {}", maybeAgent.orElse("Unknown"));
+              LOG.debug(
+                  "Connected peer id {} ({}) has agent string: {}",
+                  peerId,
+                  connection.remoteAddress(),
+                  maybeAgent.orElse("Unknown"));
```

Comment:
ooo, we should remove this log line or at least the agent string. It's not safe to log arbitrary content from external sources.  We've seen attacks where the peer agent was set to things like "Your Geth is out of date, click <malicious url> to upgrade".

Reply:
Maybe make it trace? It was already there, I've just added `peerId` and remote address

Reply:
ok, 2 votes, removing :)

Reply:
it's useful debug feature to understand what type and version of client is connecting to you